golang/cli/header.go

```diff
@@ -44,7 +44,7 @@ func prettifyHeaderData(
 // ParseHeader takes in a header and returns information about that header: digest, version, previous header hash, merkle root, timestamp, target and nonce
```

Comment:
Create `header_test.go` and move in related tests.

Reply:
these are now officially untested "example" code :)